Library/MapleBacon/MapleBacon/ImageGifExtension.swift

```diff
@@ -7,35 +7,35 @@ import ImageIO
 
 extension UIImage {
 
-    class func imageWithCachedData(data: NSData) -> UIImage? {
-        guard data.length > 0 else { return nil }
+    class func image(withCachedData data: Data) -> UIImage? {
+        guard data.count > 0 else { return nil }
         
-        return isAnimatedImage(data) ? animatedImageWithData(data) : UIImage(data: data)
+        return isAnimatedImage(data: data) ? animatedImageWithData(data: data) : UIImage(data: data as Data)
```

Comment:
I guess if we're following the new naming rules, then the method for animation should read `animatedImage(withData: data)` instead.
Don't get the point of the cast on the last expression `UIImage(data: data as Data)`??